llvm/test/DebugInfo/X86/sdag-ir-salvage.ll

```diff
@@ -7,6 +7,11 @@
 ; RUN:    -experimental-debug-variable-locations=true \
 ; RUN: | FileCheck %s --check-prefixes=CHECK,INSTRREF
 
+; RUN: llc -mtriple=x86_64-unknown-unknown -start-after=codegenprepare \
+; RUN:    -stop-before finalize-isel %s -o -  \
+; RUN:    -experimental-debug-variable-locations=false --try-experimental-debuginfo-iterators \
```

Comment:
Do we need to check `-experimental-debug-variable-locations=*true* --try-experimental-debuginfo-iterators` too? Same question for other similar tests.